fidget/src/mesh/octree.rs

```diff
@@ -128,6 +128,26 @@ impl Octree {
     ///
     /// The shape is evaluated on the region `[-1, 1]` on all axes
     pub fn build<S: Shape + Clone>(shape: &S, settings: Settings) -> Self {
+        // Transform the shape given our bounds
+        let t = settings.bounds.transform();
+        if t == nalgebra::Transform::identity() {
+            Self::build_inner(shape, settings)
+        } else {
+            let shape = shape.clone().apply_transform(t.into());
+            let mut out = Self::build_inner(&shape, settings);
+
+            // Apply the reverse transform to vertex coordinates
+            let ti = t.matrix().try_inverse().unwrap();
+            for v in &mut out.verts {
+                let p: nalgebra::Point3<f32> = v.pos.into();
+                let q = ti.transform_point(&p);
```

Comment:
I don't think using the inverse here is is correct since `t` is already the transform from `[-1, +1]` to the original bounds.

```suggestion
                let q = t.transform_point(&p);
```

Reply:
Good catch, fixed in 1d37c36336ebfb961d312057b63a97786bc99832

Reply:
Awesome! (I am unfortunately not able to mark this as resolved probably due to some repository permissions, so feel free to mark this as resolved yourself!)